api/src/pcapi/routes/public/collective/endpoints/simulate_adage_steps/bookings.py

```diff
@@ -54,3 +66,50 @@ def confirm_collective_booking(booking_id: int) -> None:
         raise ResourceNotFoundError({"code": constants.EDUCATIONAL_BOOKING_NOT_FOUND})
     except exceptions.EducationalDepositNotFound:
         raise ResourceNotFoundError({"code": "DEPOSIT_NOT_FOUND"})
+
+
+@blueprints.public_api.route("/v2/collective/bookings/<int:booking_id>/use", methods=["POST"])
+@utils.exclude_prod_environment
+@api_key_required
+@spectree_serialize(
+    api=spectree_schemas.public_api_schema,
+    on_success_status=204,
+    tags=[tags.COLLECTIVE_OFFERS],
+    resp=SpectreeResponse(
+        **(
+            http_responses.HTTP_204_COLLECTIVE_BOOKING_STATUS_UPDATE
+            | http_responses.HTTP_40X_SHARED_BY_API_ENDPOINTS
+            | http_responses.HTTP_403_COLLECTIVE_BOOKING_STATUS_UPDATE_REFUSED
+            | http_responses.HTTP_404_COLLECTIVE_OFFER_NOT_FOUND
+        )
+    ),
+)
+def use_collective_booking(booking_id: int) -> None:
+    """
+    Mock collective booking use
+
+    Use a collective booking like it would automatically be 48h after
+    the event.
+
+    Warning: not available for production nor integration environments
```

Comment:
Il faut que ce soit disponible en integration justement non ? C'est là que la feature prend tout son intérêt il me semble.

```suggestion
    Warning: not available for production nor integration environments
```

Reply:
oops... 😓